src/main/resources/templates/apidocumentation/apiDocumentationStartPage.ftl

```diff
@@ -194,7 +194,7 @@
         Email updates
       </h2>
       <p class="govuk-body">
-        You can <a class="govuk-link" href="#">sign up to receive email updates about the energy label API</a>. We'll
+        You can <a class="govuk-link" href="http://eepurl.com/h2MDDL">sign up to receive email updates about the energy label API</a>. We'll
```

Comment:
Nit but should use `https` here, even though it just redirects to a different domain

Reply:
Oh yes good spot, that's the URL Mailchimp gives you to copy so I didn't notice. Updated now